content/en/docs/Features/multi-cluster.md

```diff
@@ -3,22 +3,46 @@ title: "Multi-cluster Deployment"
 
-![Mesh list view](/images/documentation/features/multi-cluster-mesh-v1.29.png "Mesh list view")
+Each box type supports selection and provides a side-panel summary of traffic. Below we see a Bookinfo traffic graph for when Bookinfo services are deployed across the East and West clusters. The West cluster box is selected. You see traffic for all services and workloads across both clusters.
 
-## Graph View: Cluster and Namespace Boxing
+![Multi-cluster traffic graph](/images/documentation/features/multi-cluster-traffic-graph.png "Multi-cluster traffic graph")
+
+## List Views: Aggregated mesh view
+
+With a multi-cluster Kiali deployment, you can see all apps, workloads, services, and Istio config in your mesh in a single place.
+
+![Multi-cluster list pages](/images/documentation/features/multi-cluster-list.png "Multi-cluster list pages")
+
+## Detail Views: Dig into details across clusters
+
+The detail pages provide the same functionality across all clusters that you have access to for a single cluster, including viewing logs, metrics, traces, envoy details, and more.
+
+![Multi-cluster detail pages](/images/documentation/features/multi-cluster-details.png "Multi-cluster detail pages")
+
+## Overview: Cross cluster namespace info
+
+The overview page shows namespace information across all configured clusters.
+
+![Multi-cluster overview](/images/documentation/features/multi-cluster-overview.png "Multi-cluster overview")
 
-Starting in v1.8 Istio provides cluster names in the traffic telemetry for multi-cluster installations.  The Kiali graph can now use this information to better visualize clusters and namespaces.  The Display menu now offers two new options: Cluster Boxes and Namespace Boxes.  When enabled, either separately or together, the graph will generate boxes to help more easily identify the relevant nodes and edges, and to see traffic traveling between them.
+## Previous mesh support
 
-Each new box type supports selection and will provide a side-panel summary of traffic.  Below we see a Bookinfo traffic graph for when Bookinfo services are deployed across the Kukulcan and Tzotz clusters. The Kukulcan cluster box is selected. Because Kukulcan is the Kiali home cluster, you see traffic from the perspective of Kukulcan. Note that you do not see the internal traffic on Tzotz (the requests from Reviews to Ratings service).  To see traffic from the Tzotz cluster point of view, you would open a Kiali session on Tzotz, assuming you have privileges.
+Kiali has had experimental support for multi-cluster meshes since v1.29. Previously this support required multiple Kialis to run in each Kubernetes cluster and only offered a partial view of the entire mesh. Now Kiali lets you view your entire mesh from a single Kiali deployment. Much of the previous support for multi-cluster has been retained but links to external Kialis and autodiscovery of external Kialis has been removed.
```

Comment:
I feel like we should have maintained support for that model of "linked" Kiali's for when the user has multiple control planes with a Kiali on each, and may want to jump from one to the other.

Reply:
Talking about this further, generating links to external kialis should work _without_ providing access to remote clusters. By adding access to remote clusters, you are effectively enabling "multi-cluster mode" for Kiali. We can keep the existing behavior of generating links to external kialis with the secrets but by adding the secrets you will also get a "unified" view. In the future, Kiali should support generating the links without needing to grant access to remote clusters by simply providing some config values.